utility/FIOaaS/rbd/1mbps_clients.fio

```diff
@@ -0,0 +1,103 @@
+; Keep adding 1024kb/s reading clients at 4 seconds
```

Comment:
what is the purpose of this file?

Reply:
These are called .fio files used instead of long fio command

Eg: the below fio command 
fio  --filename /tmp/rep_image_rqVSD0_dir/rep_image_rqVSD0_file --size 100M --runtime 30 --time_based --name test-1 --numjobs 1 --rw write --iodepth 8 --fsync 32 --group_reporting --bs 4k

can be mapped to .fio filename as I added to the repo.